andi/values.yaml

```diff
@@ -0,0 +1,36 @@
+replicaCount: 1
+
+image:
+  repository: smartcitiesdata/andi
+  tag: 0.2.3
```

Comment:
This should be development here and then overridden with a specific tag in our deploy repo.

Reply:
Fixed